### PR TITLE
ts/parallel-axis

### PR DESCRIPTION
**Pull request by bre1470**
Migrated ParallelAxis to ES6 pattern. Replaces partially PR #13223.

### CODE REVIEW

File: ts/modules/parallel-coordinates.src.ts [hunk 1]
```diff
@@ -174,6 +163,190 @@ H.setOptions({
...
+        });
+
+        /* eslint-enable no-invalid-this */
+
+    }
+}
+
+interface ParallelAxis extends Axis {
+    chart: Highcharts.ParallelChart;
+    parallel: ParallelAxisAdditions;
```
Comment (TorsteinHonsi) [inline ts/modules/parallel-coordinates.src.ts:343]: I think this one should be called `parallelCoordinates` to avoid any future confusion.

**Comment by bre1470** [comment]
This is not the right time to work on chart compositions. Chart class is not migrated and the assembler does not support several type exports in one file.

**Comment by bre1470** [comment]
Or putted in other words, I will work on this after axis migration. :)

**Comment by TorsteinHonsi** [comment]
> Or putted in other words, I will work on this after axis migration. :)

Okay!

@pawelfus , are you okay with the latest changes? Adding the parallel coordinates stuff into an `Axis.parallelCoordinates` object?

**Pull request merged**